pina/callbacks/adaptive_refinment_callbacks.py

```diff
@@ -112,22 +114,17 @@ def _r3_routine(self, trainer):
                 pts.labels = labels
                 old_pts[location] = pts
                 tot_points += len(pts)
+            left_pts[location] = self._const_pts[location] - tot_points
+        
+            assert left_pts[location] + tot_points == self._const_pts[location] # this check improve code robustness
```

Comment:
put it as an if statement and raise a RuntimeError with the correct explanation

Reply:
Note that if I remove the computation of `left_pts`, then we go on with only the two variables  `self._const_pts` and  `old_pts`, hence we miss the third variable storing the number of points to be resampled uniformly. So it is impossible to check the  `assertion` cause we miss one information.
Anyway, the code works even in this case, we just lose the possibility of doing that check. Let me know what you want to do @dario-coscia